sass/explore.scss

```diff
@@ -1,14 +1,17 @@
-$color-primary:#9356DC;
-$color-secondary:#FF79DA;
-
 .explore {
```

Comment:
Nice file structure with SASS/ folder and css for computed files.

Reply:
Thanks !